ecs-logs-collector.sh

```diff
@@ -808,6 +808,11 @@ get_gpu_info() {
     nvidia-smi -q > "$info_system"/gpu/gpu-info.txt
   fi
 
+  # get open kernel module version
+  if [ -d /var/lib/dkms-archive/nvidia-open ]; then
```

Comment:
Q. Should we use `modinfo nvidia` in case there is more than 1 file under `/var/lib/dkms-archive/nvidia-open`?

```
[ec2-user@ip-172-31-14-70 nvidia]$ modinfo nvidia
filename:       /lib/modules/4.14.330-250.540.amzn2.x86_64/extra/nvidia.ko
firmware:       nvidia/535.129.03/gsp_tu10x.bin
firmware:       nvidia/535.129.03/gsp_ga10x.bin
alias:          char-major-195-*
version:        535.129.03
supported:      external
license:        NVIDIA
srcversion:     20B9A27F031FF0EF3B1514B
alias:          pci:v000010DEd*sv*sd*bc06sc80i00*
...
depends:        i2c-core,drm
retpoline:      Y
name:           nvidia
vermagic:       4.14.330-250.540.amzn2.x86_64 SMP mod_unload modversions 
...
```
from an EC2 gp5.xlarge instance launched with ECS AL2 GPU AMI `amzn2-ami-ecs-gpu-hvm-2.0.20231213-x86_64-ebs`.

Reply:
Because we are the only ones to [create the nvidia-open directory](https://github.com/aws/amazon-ecs-ami/blob/d8fd00ead6e8d72ce97f73b76013f93dc46fbb8c/scripts/enable-ecs-agent-gpu-support.sh#L51) we know that if this directory exists, then we've created the open kernel module tar file ourselves.

The filename `nvidia-open-535.129.03-kernel4.14.330-250.540.amzn2.x86_64-x86_64.dkms.tar.gz` gives us both the NVDIA version (535.129.03) and the Linux Kernel it was built for (kernel4.14.330-250.540.amzn2.x86_64-x86_64).

With the `ls` command we also know that we've created the .tar file.  Is there any other info from the modinfo output that you think would be useful for debugging?

Reply:
Discussed offline. Logging files under `/var/lib/dkms-archive/nvidia-open` helps us know which tar files we have, and adding `modinfo` helps us know which module is actually installed.